e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc.yml

```diff
@@ -46,6 +46,44 @@ spec:
               value: "30100"
             - name: CLIENT_PORT_MAX
               value: "30105"
+            - name: PMM_AGENT_SERVER_ADDRESS
```

Comment:
I would group environment variables differently. For example, it makes sense to group all PMM_SERVER_SERVER_XXX variables together

Reply:
ok